### ISSUE TITLE
data URL option does not work in lieu of dataSourceUrl from ChartsAPI

### ISSUE THREAD

**Issue by ktiedt** [GoogleWebComponents/google-chart#121]
As seen here: https://google-developers.appspot.com/chart/interactive/docs/reference#chartwrapperobject

Maybe its not expected to, but the assumption was, it mentions URL support for the Chart API and that was a URL for the chart API.

It would be nice if it could be supported though.

**Comment by wesalvaro** [comment]
I don't understand what you are referring to. Would you please please explain a little more or give an example?

**Comment by ktiedt** [comment]
Straight from the above link referenced in the title

```
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="content-type" content="text/html; charset=utf-8"/>
<title>Google Visualization API Sample</title>
<!--
  One script tag loads all the required libraries! Do not specify any chart types in the
  autoload statement.
-->
<script type="text/javascript"
      src='https://www.google.com/jsapi?autoload={
        "modules":[{
        "name":"visualization",
        "version":"1"
        }]
      }'></script>
<script type="text/javascript">
  google.setOnLoadCallback(drawVisualization);

  function drawVisualization() {
    var wrap = new google.visualization.ChartWrapper({
       'chartType':'LineChart',
       'dataSourceUrl':'http://spreadsheets.google.com/tq?key=pCQbetd-CptGXxxQIG7VFIQ&pub=1',
       'containerId':'visualization',
       'query':'SELECT A,D WHERE D > 100 ORDER BY D',
       'options': {'title':'Population Density (people/km^2)', 'legend':'none'}
       });
     wrap.draw();
  }
</script>
</head>
<body>
  <div id="visualization" style="height: 400px; width: 400px;"></div>
</body>
</html>
```

**Comment by wesalvaro** [comment]
OK, that's what I thought you mean, but I wasn't sure.
Correct. The element does not currently support dataSourceUrls.
I would like to switch to ChartWrapper (and already have a couple changes to make this happen) but there are some issues with the current version of the loader which create issues when switching.
As soon as they are resolved, I'll drop in my changes to use the ChartWrapper and allow you to use this feature.
Sorry for the current inconvenience.

**Comment by wesalvaro** [comment]
I should mention that you are currently able to use a Sheets DataSource through a Query object.

See [here](https://developers.google.com/chart/interactive/docs/datatables_dataviews#query) for more information.

**Comment by ktiedt** [comment]
No problem, thanks for the info! -- Honestly I would much prefer to see full chart type suite supported over this feature... This was just a surprise when I was trying to setup a demo for some co-workers.

**Comment by wesalvaro** [comment]
Yeah, for sure the DataSourceUrl should be supported. It's usage is extremely similar to the data URL in the component know but has huge fundamental differences. I'm definitely interested in twisting this element around to look almost exactly like the API for a ChartWrapper soon.

**Comment by wesalvaro** [comment]
I've created a PR that adds a Query element.
This way, you could do:
```js
<google-chart-query url="..." data="{{data}}">
<google-chart data="[[data]]">
```
